joinfaces-dependencies/pom.xml

```diff
@@ -314,6 +318,43 @@
                 <scope>runtime</scope>
             </dependency>
 
+            <dependency>
+                <groupId>com.github.adminfaces</groupId>
+                <artifactId>admin-template</artifactId>
+                <version>${adminfaces.version}</version>
+                <scope>compile</scope>
+                <exclusions>
+                    <exclusion>
+                        <groupId>org.primefaces</groupId>
+                        <artifactId>primefaces</artifactId>
+                    </exclusion>
+                    <exclusion>
+                        <groupId>org.omnifaces</groupId>
+                        <artifactId>omnifaces</artifactId>
+                    </exclusion>
+                    <exclusion>
+                        <groupId>org.adminfaces</groupId>
+                        <artifactId>admin-theme</artifactId>
+                    </exclusion>
+                </exclusions>                
+            </dependency>
+            <dependency>
+                <groupId>com.github.adminfaces</groupId>
+                <artifactId>admin-theme</artifactId>
+                <version>${adminfaces.version}</version>
+                <scope>compile</scope>
+            </dependency>
+            <dependency>
+                <groupId>org.hibernate.javax.persistence</groupId>
+                <artifactId>hibernate-jpa-2.1-api</artifactId>
```

Comment:
If this is the same jar also used by `spring-boot-starter-data-jpa`, should'nt the dependencyManagement for this jar already contained in spring-boot itself? 🤔

Reply:
No, because we don't pull `spring-boot-starter-data-jpa`. Should we pull it? I think it contains a lot of other things we do not care about.